From 4d442a43d684a6d7482cb3b7b4d3cdee003df26f Mon Sep 17 00:00:00 2001
From: Joachim Falk <joachim.falk@gmx.de>
Date: Thu, 13 Oct 2011 19:44:06 +0200
Subject: [PATCH 2/7] Add RH patch tigervnc11-rh645755.patch fix signal interrupted read

The Xvnc server randomly refused connections when the reading of the
password file (provided when starting Xvnc with the "-PasswordFile"
option) was interrupted by a signal. With this update, the loading of a
password file continues after an interrupt signal is issued and
connections are no longer refused. (BZ#645755)
---
 common/rfb/SSecurityVncAuth.cxx |   16 +++++++++++++++-
 1 files changed, 15 insertions(+), 1 deletions(-)

--- a/common/rfb/SSecurityVncAuth.cxx
+++ b/common/rfb/SSecurityVncAuth.cxx
@@ -31,6 +31,7 @@
 #include <rfb/Exception.h>
 #include <string.h>
 #include <stdio.h>
+#include <errno.h>
 extern "C" {
 #include <rfb/d3des.h>
 }
